xonsh/environ.py

```diff
@@ -191,6 +194,7 @@ def xonshconfig(env):
     'PUSHD_SILENT': False,
     'RAISE_SUBPROC_ERROR': False,
     'RIGHT_PROMPT': '',
+    'DYNAMIC_CWD_WIDTH': '20',
```

Comment:
What should be the string representation of infinite ? +INF 

Maybe it would not be good to replace the default cwd because this one if more complex to compute maybe not everyone is willing to pay this price